### PR TITLE
fix dummy library generation 

### PR DESCRIPTION
**Pull request by shareefalis**

### CODE REVIEW

**Comment by ianblenke** [comment]
Not sure if you want me to merge this or not yet.

**Comment by ianblenke** [comment]
Travis-CI is failing to build this. Shareef warned me not to merge it yet.

**Comment by shareefalis** [comment]
DO NOT MERGE!

**Comment by shareefalis** [comment]
dummy library fails to compile this, also you need to have  an updated linphone submodules..

File: linphone.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -2487,11 +2491,14 @@
 				220FAD3910765B400068D98F /* libspeexdsp.a in Frameworks */,
 				226183AE1472527D0037138E /* libsrtp.a in Frameworks */,
 				22276E8513C73D3100210156 /* libswscale.a in Frameworks */,
+				63D7215D1B7394D200D70E65 /* libtunnel.a in Frameworks */,
```
Comment (ianblenke) [inline linphone.xcodeproj/project.pbxproj:2494]: Should there be a tunnel.a here? Mostly concerned about introducing a private dependency

Comment (shareefalis) [inline linphone.xcodeproj/project.pbxproj:2494]: belledonne added it as a dummy library compile.

Comment (shareefalis) [inline linphone.xcodeproj/project.pbxproj:2494]: belledonne added reading from a xcode file and extracting it from there.. that where all the trouble began. I missed one line  when merging the new stuff  https://github.com/shareefalis/ace-iphone/commit/fa222a3ca1cd9e8372fef9b87e52e34191495ff6

**Comment by ianblenke** [comment]
PR26 fixes this build problem. It does build correctly with travis

Going to +1 this one, and merge that one.

**Pull request merged**